src/OrleansCodeGenerator/RoslynCodeGenerator.cs

```diff
@@ -484,6 +480,36 @@ private static void RecordType(Type type, Module module, Assembly targetAssembly
         }
 
         /// <summary>
+        /// Get types which have corresponding generated classes.
+        /// </summary>
+        /// <returns>Types which have corresponding generated classes marked.</returns>
+        private static HashSet<Type> GetTypesWithGeneratedSupportClasses()
+        {
+            // Get assemblies which contain generated code.
+            var all =
+                AppDomain.CurrentDomain.GetAssemblies()
+                    .Where(_ => _.GetCustomAttribute<GeneratedCodeAttribute>() != null)
+                    .SelectMany(_ => _.DefinedTypes);
```

Comment:
Hmmm, I think there might be an issue with this logic: `assembly.GetCustomAttribute<GeneratedCodeAttribute>()` should always return a non-null array, it's just that it's empty if the attribute is not there...

BTW, I know this was just copied over, but if you can, please use an argument name in your lambda expression. Convention has it that if you use `_` it's because you don't care about it's value at all and you are just receiving it because the method signature expects you to.